src/libxt_TS3INIT_SET_COOKIE.c

```diff
@@ -85,11 +85,11 @@ static void ts3init_set_cookie_tg_save(const void *ip, const struct xt_entry_tar
     const struct xt_ts3init_set_cookie_tginfo *info = (const void *)target->data;
     if (info->specific_options & TARGET_SET_COOKIE_ZERO_RANDOM_SEQUENCE)
     {
-        printf("--zero-random-sequence ");
+        printf(" --zero-random-sequence ");
     }
     if (info->specific_options & TARGET_SET_COOKIE_RANDOM_SEED_FROM_ARGUMENT)
     {
-        printf("--random-seed ");
+        printf(" --random-seed ");
```

Comment:
please remove printf(" ") at the end of the loop. add a space to --random-seed-file and remove the trailing space there, and finally remove trailing space at  printf(" -j TS3INIT_SET_COOKIE ");